modules/ROOT/pages/viewing-log-data.adoc

```diff
@@ -2,53 +2,40 @@
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
-:keywords: cloudhub, logging, enhanced log management, r44, arm, runtime manager
-
-image:logo-cloud-active.png[link="/runtime-manager/deployment-strategies", title="CloudHub"]
-image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies", title="Hybrid Deployment"]
-image:logo-server-disabled.png[link="/runtime-manager/deployment-strategies", title="Anypoint Platform Private Cloud Edition"]
-image:logo-pcf-disabled.png[link="/runtime-manager/deployment-strategies", title="Pivotal Cloud Foundry"]
 
 CloudHub provides access to log data that includes deployment messages and events for each worker.
 
-
 [NOTE]
 ====
-Logs and diagnostics are currently available only for applications deployed to CloudHub workers. In cases where applications are deployed using the Runtime Manager local server, you can send data xref:sending-data-from-arm-to-external-analytics-software.adoc[to External Analytics Software]. See xref:deployment-strategies.adoc[Deployment Strategies] and xref:monitoring.adoc[Monitoring Applications] for more details.
+Logs and diagnostics are currently available only for applications deployed to CloudHub workers. In cases where applications are deployed to a local server, you can send data to external analytics software. 
 
 Applications deployed to Pivotal Cloud Foundry (PCF) don't expose any log data through Runtime Manager, but you can see event logs on Pivotal's UI.
 ====
 
 
 == View Logs
 
-. In Runtime Manager, click *Applications* in the left menu, then click the application that is deployed to CloudHub to open its management options.
-. In the left menu, click *Logs*. The application logs are displayed.
-+
-//image::elmstart.png[ELMStart]
+View logs for applications deployed to CloudHub.
 
+. In Runtime Manager, click *Applications* in the left menu, then click the application that is deployed to CloudHub to open the management panel on the right. 
```

Comment:
Is the "left menu" the navigation pane?

GLOBALLY replace "...blah, then" with "blah, and then" or "blah; then, "
A comma followed by "then" is not a conjunction.

Reply:
No semicolons please. They rarely belong in technical writing.

Reply:
Interesting. My previous editor would disagree with the ", and then" construction - she'd say either the and or the then :)